fem/bilininteg_mass.cpp

```diff
@@ -216,11 +216,10 @@ static void PAMassApply2D(const int NE,
    auto y = Reshape(_y.ReadWriteAccess(), D1D, D1D, NE);
    MFEM_FORALL(e, NE,
    {
-      const int D1D = T_D1D ? T_D1D : d1d; // nvcc workaround
+      const int D1D = T_D1D ? T_D1D : d1d;
       const int Q1D = T_Q1D ? T_Q1D : q1d;
```

Comment:
Is there a reason to remove the comment? I thought it conveyed useful information...

Reply:
I agree. If this was my edit, it was not intentional.

Reply:
I think it's me, sorry about that.